Manuals/User manual.md

```diff
@@ -0,0 +1,169 @@
+
+![image](image/Protagonist.png)
+### hub
+
+**The base camp**
+
+The hub is the first place you will discover when starting a new game. This place is the bridge between every zone due to the teleporter allowing you to travel through the map. You will also find a map of the planet and the scientist waiting to analyze your discoveries.
+
+![image](image/Hub.png)
+### narrative-lore
+
+**Narrative**
+
+By the 2040s, climate impacts from unchecked industrial growth were severe, despite advances in clean energy and technology. Efforts to reverse the damage, like CO2 scrubbers, came too late. By 2130, runaway climate change made Earth uninhabitable, with ecosystems collapsing and humanity wiped out—a stark warning of the cost of neglect and hubris.
+
+**Lore**
+
+You will find pieces of lore in the different zones. As they are hidden you will have to find them to learn more about the world you explore. The importance of the lore lies in the understanding you will have of the importance of ecology.
+
+### mechanics
+
+**Movement**
+
+Top-Down View:
+
+Gravity is disabled in this view to create a grounded exploration experience, allowing the player to focus on navigation and interaction with the environment.
+
+The player can move freely in all four directions—north, south, east, and west—offering a broad field for outdoor exploration and precise control.
+
+![image](image/Top%20view.png)
+
+Side-Scroller View:
+
+Gravity is enabled, introducing jumping mechanics for indoor exploration. This vertical movement adds depth to indoor navigation and allows for platforming challenges.
+
+Limited to horizontal movement (left and right) to align with the side-scrolling perspective. Obstacles and gaps may require jumping to overcome.
+
+![image](image/Side%20view.png)
+
+Objects and Item Interaction
+
+Objects within both perspectives are non-physics-based and remain static until interacted with. This design ensures that items are easy to locate and interact with without requiring complex manipulation.
+
+Items can be collected when players approach and interact with them. This includes key items, collectibles, and objects necessary for puzzle-solving.
+
+Actions and Interactions
+
+Interact: The player can interact with various elements within the game, including inscriptions, levers, collectible items, and locked doors. These interactions will either provide information, unlock new areas, or progress objectives.
+
+Call: During certain moments, the player can call the scientist NPC to get hints or directives from him, obtaining guidance for navigating the game world or solving puzzles. This mechanic adds a sense of companionship and progression without disrupting gameplay.
+
+
+
+### team-6-credits
+
+[PM] - Mathias GAGNEPAIN <br>
+[PGM]- Enzo GUILLOUCHE <br>
+[TL] - Evan UHRING <br>
+[QA] - Tino GABET <br>
+[SE] - Michel RIFF <br>
+[SE] - Loïc NOGUES <br>
+[TW] - Axel DAVID
+
+## Thank you for reading !
+## Have a nice game !
```

Comment:
idk if game is the appropriate word

Reply:
Yes because the person reading this manual is going to play a game